features/ftue/impl/src/main/kotlin/io/element/android/features/ftue/impl/state/DefaultFtueService.kt

```diff
@@ -90,34 +105,19 @@ class DefaultFtueService @Inject constructor(
             } else {
                 getNextStep(FtueStep.AnalyticsOptIn)
             }
-            FtueStep.AnalyticsOptIn -> {
-                updateState()
```

Comment:
This was a fix for application without analytics (see https://github.com/element-hq/element-x-android/pull/3337). So maybe restore?

Reply:
In any case, I'll leave it there just in case, since this might be caused by something else triggering the `moveToNextStepIfNeeded()` function and we might remove that calling site in the future and come back to the bug described there.